bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -76,7 +76,7 @@ spec:
 {{- include "mariadb-galera.imagePullSecrets" . | indent 6 }}
       {{- if .Values.extraInitContainers }}
       initContainers:
-{{ tpl .Values.extraInitContainers . | indent 6}}
+{{ tpl (toYaml .Values.extraInitContainers) . | indent 6 }}
```

Comment:
```diff
- {{ tpl (toYaml .Values.extraInitContainers) . | indent 6 }}
+ {{ tpl (toYaml .Values.extraInitContainers) . | indent 8 }}
```

Reply:
Why? Is this you preference? It works with both.
Indent 6:
```
$ helm template -x templates/statefulset.yaml . | grep initContainer -A 6                                                                         
      initContainers:
      - command:
        - do
        - something
        image: bitnami/minideb:stretch
        name: do-something
```
Indent 8:
```
helm template -x templates/statefulset.yaml . | grep initContainer -A 6
      initContainers:
        - command:
          - do
          - something
          image: bitnami/minideb:stretch
          name: do-something
```

Reply:
Oh, okay no worries, I didn't think about the array.